arborist/server_test.go

```diff
@@ -3018,6 +3540,42 @@ func TestServer(t *testing.T) {
 					msg := fmt.Sprintf("got response body: %s", w.Body.String())
 					assert.Equal(t, []string{resourcePath}, result.Resources, msg)
 				})
+
+				// Setup for GET_noDuplicatedMappings:
+				// Add the policies in the `anonymous` group to the user.
+				// Adding the policies of the `anonymous` group to the user also adds
+				// the resources of the `anonymous` group to the user.
+				for _, policy := range anonymousPolicies {
+					grantUserPolicy(t, username, policy.Name)
+				}
```

Comment:
why not just put this in `GET_noDuplicatedMappings`?

Reply:
I think I left it outside the function because it's a side effect that doesn't get undone after GET_noDuplicatedMappings, so I thought it was less confusing to have it outside the function. But now that you're pointing it out I think it would be way better to move it inside the function and tear it down afterward by removing the user policy, so I think I'll make that change.